python/paddle/fluid/__init__.py

```diff
@@ -60,6 +60,7 @@
     'io',
     'initializer',
     'layers',
+    'transpiler'
```

Comment:
It seems that a comma is missing here...

Reply:
Sorry for the mistake... will fix it.